server/src/main/java/io/druid/server/AsyncQueryForwardingServlet.java

```diff
@@ -274,7 +274,7 @@ public void onComplete(Result result)
                         "query/time",
                         requestTime,
                         "success",
-                        true
+                        result.isSucceeded()
```

Comment:
I think this would cause requests to fail more often, wouldn't it be better if requests were always successful?